CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 * [CHANGE] Fix incorrectly named `cortex_cache_fetched_keys` and `cortex_cache_hits` metrics. Renamed to `cortex_cache_fetched_keys_total` and `cortex_cache_hits_total` respectively. #4686
 * [CHANGE] Enable Thanos series limiter in store-gateway. #4702
 * [CHANGE] Distributor: Apply `max_fetched_series_per_query` limit for `/series` API. #4683
+* [CHANGE] Re-enable the `proxy_url` option for receiver configuration. #4680
```

Comment:
In change log, the convention is to put the PR number instead of the issue number. May I ask you to kindly update it? Thanks!

Reply:
Of course, done!